src/style.css

```
@@ -0,0 +1,91 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+  --theme-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.product-container {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  padding: 29px 14px 14px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-container__img {
+  display: block;
+  margin: 0 auto 45px;
+  width: 160px;
+  height: 134px;
+}
+
+.product-container__card-header {
+  font-weight: 500;
+  line-height: 15px;
+  font-size: 12px;
+  margin: 0 auto 6px;
+}
+
+.product-container__code {
+  padding-bottom: 16px;
```

```suggestion
  padding-bottom: 15px;
```
One of the reasons why your tests are failed